src/docs/options.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Options'
+---
+
+Options are a way to store generic system-wide configuration. They serve a similar purpose to configuration files, but they are backed by a database, so it's possible to change them at runtime without a deploy. This makes options well-suited for rates, quotas, and limits.
+
+## Adding New Options
+
+To add a new system option, add it to [`sentry/options/defaults.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/options/defaults.py) by registering it. e.g.,
+
+```python
+register("performance.some-feature-rate", default=0.0)
+```
+
+### Tips
+
+- namespace your option (e.g., `feature-scope.feature-name`)
+- colocate your new option with related existing options
+- use dashes and not underscores
+- always set a default value with `default=`
+- the value of an option can be any pickleable value
+
+<Alert level="warning">
+  Always split up registering the new option and the code that relies on the new option. Register new options in their own pull requests, ahead of the code changes.
+</Alert>
+
+## Using Options
+
+To check the value of an option, import the `options` module and use the `get` method. e.g.,
+
+```python
+from sentry import options
+feature_rate = options.get("performance.some-feature-rate")
+```
+
+## Setting Options
+
+Options are stored in the database and are cached. You can change the value of an option using `sentry shell`, or by using the management UI.
+
+### Sentry Shell
+
+You can set option values using `sentry shell`. e.g.,
+
+```bash
+sentry shell
+```
+
+then,
+
+```python
+from sentry import options
+options.set("performance.some-feature-rate", 0.01)
+```
+
+<Alert level="info">
+If you do not have access to the shell, you'll need to contact OPS to set the option value for you.
+</Alert>
+
+### Management UI
+
+If you expect to frequently update your option, you can make it editable in the management UI. The `/admin/settings` page in the Sentry app has inputs for a small set of the available options. You can submit a pull request to add your option to the UI by [updating the corresponding React view](https://github.com/getsentry/sentry/blob/master/static/app/views/admin/options.tsx).
+
+## Using Options for Feature Rollout
+
+If you're working on a system-wide feature, you may choose to use options for your rollout instead of feature flags. Unlike feature flags, options don't allow for easy segmentation, but they are performance, stable, and simple. e.g.,
```

Comment:
```suggestion
If you're working on a system-wide feature, you may choose to use options for your rollout instead of feature flags. Unlike feature flags, options don't allow for easy segmentation, but they are performant, stable, and simple to implement. e.g.,
```

Reply:
Thank you!